homeassistant/components/minecraft_server/strings.json

```
@@ -5,7 +5,7 @@
         "title": "Link your Minecraft Server",
         "description": "Set up your Minecraft Server instance to allow monitoring.",
         "data": {
-          "name": "Name",
+          "name": "[%key:common::config_flow::data::name%]",
```

I would recommend to revert this line so this can be merged
```suggestion
          "name": "Name",
```

---

I've rebased on latest dev branch. The build should pass now.